### ISSUE TITLE
Creating an entity with a lookup to another entity that doesn

### ISSUE THREAD

**Issue by aleksanderwaagr** [jordimontana82/fake-xrm-easy#204]

**Comment by aleksanderwaagr** [comment]
Real CRM doesn't allow referencing entities that doesn't exist. I'm trying to write a migration tool, so the order data is created is important. I tried adding validation to entity reference attributes on create, but it results in about one third of all unit tests failing. Does this mean this behavior is a deliberate choice?

**Comment by jordimontana82** [comment]
Hi Aleksander,

Thanks for raising the issue. Indeed, that's not being validated yet. We could include it in a future release.

**Comment by aleksanderwaagr** [comment]
Great! I see that initialization and "create" use very overlapping logic. I guess they should be separated a bit so that "create" can have validation but it would be nice if initialization can skip it.

**Comment by jordimontana82** [comment]
Good idea. Yep. It could be enabled based on a app.config parameter...
